backend/src/main.py

```diff
@@ -190,6 +190,7 @@ def create_source_node_graph_url(uri, db_name, userName, password, source_url, m
            job_status = "Completed"
            return create_api_response(job_status,data='Unknown URL')
     except Exception as e:
+        source_type = ''
         job_status = "Failed"
         error_message = str(e)
         logging.exception(f'Exception Stack trace:')
```

Comment:
exception is not actually logged or exception message returned

Reply:
@jexp Exception logging in each function